src/main/java/net/minecraftforge/fluids/FluidRegistry.java

```diff
@@ -356,6 +356,21 @@ public static String getDefaultFluidName(Fluid key)
         return name;
     }
 
+    @Nullable
+    public static String getModId(@Nullable FluidStack fluidStack)
+    {
+        if (fluidStack != null)
```

Comment:
Would it be a good idea to use `Optional` instead of `null`? All these nullguards are itching for some `flatMap` magic.